profiles/lib/tags.lua

```diff
@@ -14,13 +14,23 @@ function Tags.get_forward_backward_by_key(way,data,key)
   local forward = way:get_value_by_key(key .. ':forward')
   local backward = way:get_value_by_key(key .. ':backward')
 
-  if forward and backward then
-    return forward, backward
+  if not forward or not backward then
+     local common = way:get_value_by_key(key)
+
+     if (data.oneway) then
+        if data.is_forward_oneway then
+           forward = forward or common
+        end
+        if data.is_reverse_oneway then
```

Comment:
Is this an added bug fix for the profile? It might make sense to capture that separately.

Reply:
I found some bugs and will open a separate PR for review, namely here is a fix for double counting of lanes in case of `oneways`.